client/stylesheets/sass/Academics.scss

```diff
@@ -0,0 +1,42 @@
+.header {
+  width: 100%;
+  height: 200px;
+  background-color: $blue;
+  color: white;
+  margin: 0 auto;
+  text-align: center;
+
+  .header_text {
+    @include vertical-align();
+  }
+}
+
+.sub_header {
+  width: 90%;
+  height: 60px;
+  background-color: $light-grey;
+  color: black;
+  text-align: center;
+  margin: 30px auto 0;
+
+  .header_text {
+    @include vertical-align();
+  }
+}
+
+.triangle {
+  width: 0;
+  height: 0;
+  margin: 0 auto;
+  border-style: solid;
+  border-width: 39.0px 22.5px 0 22.5px;
+  border-color: $light-grey transparent transparent transparent;
+  line-height: 0;
+  _border-color: $light-grey #000 #000 #000;
+  _filter:progid:DXImageTransform.Microsoft.Chroma(color='#000000');
```

Comment:
In the mockups, the subheaders have a little triangle attached to them so this is just some css to make those triangles
![image](https://cloud.githubusercontent.com/assets/8943563/10470538/bf69eb8e-71c2-11e5-99d2-5ddd555ff2b7.png)